packages/@lwc/integration-karma/test/light-dom/slotting/index.spec.js

```diff
@@ -131,29 +131,4 @@ describe('Slotting', () => {
             expect(commentNodes.length).toBe(0); // old implementation does not use fragments, just flattening
         }
     });
-
-    // TODO [#3974]: remove temporary logic to support v5 compiler + v6+ engine
-    if (process.env.FORCE_LWC_V5_COMPILER_FOR_TEST) {
-        describe('DISABLE_TEMPORARY_V5_COMPILER_SUPPORT', () => {
-            beforeEach(() => {
-                setFeatureFlagForTest('DISABLE_TEMPORARY_V5_COMPILER_SUPPORT', true);
-            });
-
-            afterEach(() => {
-                setFeatureFlagForTest('DISABLE_TEMPORARY_V5_COMPILER_SUPPORT', false);
-            });
-
-            it('should not render slots properly', () => {
-                const nodes = createTestElement('x-default-slot', BasicSlot);
-
-                expect(Array.from(nodes['x-container'].children)).toEqual([
-                    nodes['container-upper-slot-default'],
-                    nodes['upper-text'],
-                    nodes['default-text'],
-                    nodes['lower-text'],
-                    nodes['container-lower-slot-default'],
-                ]);
-            });
-        });
-    }
```

Comment:
In the [PR](https://github.com/salesforce/lwc/pull/3973) to support LWC v5 there was a [test case removed in favor of this one](https://github.com/salesforce/lwc/pull/3973/files#diff-17fa10a6f951fe391e08cab4bf7b943fe4d914c083a104de60eb85d33dd4c58dR23-R24), should we move it back?

Reply:
Whoops, missed that! Added back: 7fe7dbd64